clients/unshaded/src/test/java/tachyon/client/block/BufferedBlockInStreamTest.java

```diff
@@ -21,24 +21,39 @@
 
 import tachyon.util.io.BufferUtils;
 
+/**
+ * Tests for the {@link BufferedBlockInStream} class.
+ */
 public class BufferedBlockInStreamTest {
   private static final long BLOCK_LENGTH = 100L;
 
   private TestBufferedBlockInStream mTestStream;
 
+  /**
+   * Sets up the stream before a test runs.
+   */
   @Before
   public void before() {
     mTestStream = new TestBufferedBlockInStream(1L, 0, BLOCK_LENGTH);
   }
 
+  /**
+   * Verifies byte by byte read is equal to increasing byte array.
```

Comment:
Thanks for the comment. I struggled with that a little bit, because this was originally a comment in the test itself. I rewrote it in an update for this PR.